src/components/MenuItem/MenuItem.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import "./MenuItem.scss";
+
+export const MenuItem = (props) => {
+    return (
+        <div className="MenuItem">
+            <div className="crossIcon">
```

Comment:
This is wrong it should be `MenuItem__crossIcon` as it is you are not acutaly using the css written in your file